src/providers/chatterino/ChatterinoBadges.cpp

```diff
@@ -13,7 +13,7 @@
 #include <map>
 
 namespace chatterino {
-void ChatterinoBadges::initialize(Settings &settings, Paths &paths)
+void ChatterinoBadges::initialize(Settings &, Paths &)
```

Comment:
Here it's obvious what the variables are so I'm fine with this. In some other cases where there's only a `bool` you have no idea what it is without knowing the name of the variable.